### DISCUSSION TITLE
AfterClose event

### DISCUSSION THREAD

**Discussion by oshumelka** [fancyapps/ui#453] [Q&A]
Hello! 
I need to use AfterClose event (like in v3). How can I do that?
Looks like destroy event (Closing is complete) not fit for this? [jsfiddle](https://jsfiddle.net/shumilka/68e0jxh9/)

**Comment by fancyapps** [comment]
Hi,

Simply use `*` to see all available events and then choose the needed - https://jsfiddle.net/j9t36ybq/
